tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -72,8 +75,10 @@ export class Ide {
         } catch (err) {
             if (err instanceof error.TimeoutError) {
                 Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out.`);
+            } else {
+                Logger.error(`Waiting for task notification failed after ${timeout} timeout.`)
```

Comment:
In context of previous "if" statement it is not logical. If it is timeout error it will be cached in the previous "if". It may be confusing.